src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimeline.tsx

```diff
@@ -94,20 +91,22 @@ function BreakpointTimeline({
       seek(event.point, hoveredTime, false);
     }
   };
-  const onMouseMove = (e: React.MouseEvent) => {
+  const onMouseMove = (event: MouseEvent) => {
     const { beginTime, endTime } = zoomRegion;
-    const { left, width } = e.currentTarget.getBoundingClientRect();
-    const clickLeft = e.clientX;
+    const { left, width } = event.currentTarget.getBoundingClientRect();
+    const clickLeft = event.clientX;
 
     const clickPosition = Math.max((clickLeft - left) / width, 0);
     const time = Math.ceil(beginTime + (endTime - beginTime) * clickPosition);
     setHoveredTime(time);
   };
-  const onMouseLeave = (e: React.MouseEvent) => setHoveredTime(null);
+  const onMouseLeave = () => setHoveredTime(null);
 
   const hoverPercent = `${getVisiblePosition({ time: hoveredTime, zoom: zoomRegion }) * 100}%`;
   const percent = getVisiblePosition({ time: currentTime, zoom: zoomRegion }) * 100;
 
+  // TODO [bvaughn] UnloadedRegions should be loaded + focused
```

Comment:
is this todo okay?

Reply:
Ooops. Forgot to delete this comment. Yeah, I already did that. It's what was fixed here: https://github.com/replayio/devtools/pull/7643/files/606ba4950b4c24a83cf748ef3af266aa06b71afd#diff-c9da310bb17d439da080e71962563ba5a48b4740dd69f5243d3f8c33fff2f8d7